docs/reference/buildx_history_export.md

```diff
@@ -15,3 +15,31 @@ Export a build into Docker Desktop bundle
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Export one or more build records to `.dockerbuild` archive files. These archives
+contain metadata, logs, and build outputs, and can be imported into Docker
+Desktop or shared across environments.
+
+## Examples
+
+### Export a single build to a custom file
+
+```console
+docker buildx history export mybuild --output mybuild.dockerbuild
```

Comment:
`mybuild` argument is not correct, it should be a build ref not a name like ones you see from `history ls` command:

```
$ docker buildx history ls
BUILD ID                    NAME                                   STATUS      CREATED AT    DURATION   
qu2gsuo8ejqrwdfii23xkkckt   .dev/2850                              Completed   3 days ago    1.4s       Open
qsiifiuf1ad9pa9qvppc0z1l3   .dev/2850                              Completed   3 days ago    1.3s       Open
```

```
$ docker buildx history export --output foo.dockerbuild qu2gsuo8ejqrwdfii23xkkckt
```